From: olh@suse.de
Subject: add missing include

drivers/scsi/qla2xxx/qla_rscn.c:334: error: implicit declaration of function 'fc_remote_port_unblock'

Index: linux-2.6.13/drivers/scsi/qla2xxx/qla_rscn.c
===================================================================
--- linux-2.6.13.orig/drivers/scsi/qla2xxx/qla_rscn.c
+++ linux-2.6.13/drivers/scsi/qla2xxx/qla_rscn.c
@@ -17,6 +17,7 @@
  *
  */
 #include "qla_def.h"
+#include <scsi/scsi_transport_fc.h>
 
 /**
  * IO descriptor handle definitions.
